pkg/portal/middleware/aad.go

```diff
@@ -124,6 +124,8 @@ func NewAAD(log *logrus.Entry,
 	a.store.Options.SameSite = http.SameSiteLaxMode
 
 	unauthenticatedRouter.NewRoute().Methods(http.MethodGet).Path("/callback").Handler(Log(env, audit, baseAccessLog)(http.HandlerFunc(a.callback)))
```

Comment:
I think we should change `callback` to `/api/callback` but we can do this later :)